splinterd/src/config/mod.rs

```diff
@@ -134,3 +134,305 @@ impl Config {
         self.heartbeat_interval
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    use std::fs;
+
+    /// Paths to existing example config toml files from the top-level Splinterd directory.
+    static TEST_TOML1: &str = "sample_configs/splinterd.toml.example";
+    /// This toml file is only used in testing the TomlConfig module and therefore only used with
+    /// the `toml-config` feature.
+    #[allow(dead_code)]
+    static TEST_TOML2: &str = "sample_configs/splinterd.toml.example2";
+
+    /// Values present in the existing example config toml files.
+    static STORAGE: &str = "yaml";
+    static TRANSPORT: &str = "tls";
+    static CA_CERTS: &str = "certs/ca.pem";
+    static CLIENT_CERT: &str = "certs/client.crt";
+    static CLIENT_KEY: &str = "certs/client.key";
+    static SERVER_CERT: &str = "certs/server.crt";
+    static SERVER_KEY: &str = "certs/server.key";
+
+    /// Config values unique to the TEST_TOML1 file.
+    static SERVICE_ENDPOINT: &str = "127.0.0.1:8043";
+    static NETWORK_ENDPOINT: &str = "127.0.0.1:8044";
+    static NODE_ID: &str = "012";
+
+    /// Config values unique to the TEST_TOML2 file.
+    /// These values are only used in testing the TomlConfig module and therefore only used with
+    /// the `toml-config` feature.
```

Comment:
These should be guarded by the toml-config feature then instead of marked dead-code.

Reply:
Addressed